WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -209,7 +209,43 @@ - (void)handleApplicationDidBecomeActive:(NSNotification *)notification
     [self.view layoutIfNeeded];
 }
 
+#pragma mark - Tracking methods
 
+-(void)trackCommentsOpened {
+    NSMutableDictionary *properties = [NSMutableDictionary dictionary];
+    properties[WPAppAnalyticsKeyPostID] = self.post.postID;
+    properties[WPAppAnalyticsKeyBlogID] = self.post.siteID;
+    [WPAppAnalytics track:WPAnalyticsStatReaderArticleCommentsOpened withProperties:properties];
+}
+
+-(void)trackCommentLikedOrUnliked:(Comment *) comment {
+    ReaderPost *post = self.post;
+    WPAnalyticsStat stat = comment.isLiked
+    ? WPAnalyticsStatReaderArticleCommentLiked
```

Comment:
tiniest nitpick: i think some indentation here would help with visually parsing the ternary operator here.

but i have a very personal thing against them, so it might be just me :)